### DISCUSSION TITLE
通过addShape添加的图形, 无法添加节点通用属性

### DISCUSSION THREAD

**Discussion by zmcode** [antvis/G6#5213] [General]
![image](https://github.com/antvis/G6/assets/48686959/58156a66-b957-4263-b33a-1a395e4f3615)

可以看到只支持增加样式,但是通用属性无法添加, 例如我要添加一个rect,加一个label, 但是无法添加, 只能在创建一个text的shape

https://g6.antv.antgroup.com/manual/middle/elements/nodes/default-node#%E8%8A%82%E7%82%B9%E7%9A%84%E9%80%9A%E7%94%A8%E5%B1%9E%E6%80%A7

**Comment by Aarebecca** [comment]
你可以先通过 addShape 创建一个 rect，然后在该 rect 上调用 addShape 来创建一个 label
